docs/internals/api/index.txt

```diff
@@ -0,0 +1,13 @@
+============
+Django's API
+============
+
+This section of Django's documentation describes Django's internals from a
+algorithmic level than other areas of the documentation. It should not be seen
```

Comment:
should be '.. algorithmic level as opposed to other areas of the documentation.' ?